src/popup/router/pages/TransactionDetails.vue

```diff
@@ -0,0 +1,182 @@
+        </InfoBox>
+        <InfoBox
+          :value="hash"
+          :label="$t('pages.transactionDetails.hash')"
+          class="span-2-columns"
+          small
+        >
+          <CopyButton slot="label" :value="hash" message="Hash copied" />
+        </InfoBox>
+        <InfoBox
+          :value="microTime | formatDate"
+          :secondary="microTime | formatTime"
+          :label="$t('pages.transactionDetails.timestamp')"
+        />
+        <InfoBox
+          v-if="blockHeight"
+          :value="blockHeight"
+          :label="$t('pages.transactionDetails.blockHeight')"
+        />
+        <InfoBox v-if="tx.gas" :value="tx.gas" :label="$t('pages.transactionDetails.gas')" />
+        <InfoBox v-if="tx.gasPrice" :label="$t('pages.transactionDetails.gasPrice')">
+          <TokenAmount slot="value" :amount="tx.gasPrice" symbol="ættos" hideFiat />
+        </InfoBox>
+        <InfoBox :label="$t('pages.transactionDetails.amount')">
+          <TokenAmount slot="value" :amount="amount" :symbol="symbol" hideFiat />
+        </InfoBox>
+        <InfoBox v-if="tx.nonce" :value="tx.nonce" :label="$t('pages.transactionDetails.nonce')" />
+        <InfoBox
+          v-if="pending"
+          :value="$t('pages.transactionDetails.pending')"
+          :label="$t('pages.transactionDetails.status')"
+          highlight
+        />
+        <InfoBox v-if="tx.fee" :label="$t('pages.transactionDetails.fee')" class="span-2-columns">
+          <TokenAmount slot="value" :amount="tx.fee" symbol="ættos" hideFiat />
+        </InfoBox>
+      </div>
+      <div class="explorer">
+        <LinkButton @click="openExplorer">
+          <AnimatedPending v-if="pending" />
+          <BlockIcon v-else />
+          {{ $t('pages.transactionDetails.explorer') }}
+        </LinkButton>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { formatDate, formatTime } from '../../utils';
+import openUrl from '../../utils/openUrl';
+import TokenAmount from '../components/TokenAmount';
+import InfoBox from '../components/InfoBox';
+import LinkButton from '../components/LinkButton';
+import CopyButton from '../components/CopyButton';
+import AnimatedPending from '../../../icons/animated-pending.svg?vue-component';
+import BlockIcon from '../../../icons/block.svg?vue-component';
+
+export default {
+  name: 'TransactionDetails',
+  components: {
+    TokenAmount,
+    InfoBox,
+    LinkButton,
+    CopyButton,
+    AnimatedPending,
+    BlockIcon,
+  },
+  props: {
+    tx: { type: Object, required: true },
+    hash: { type: String, required: true },
+    microTime: { type: Number, required: true },
+    blockHeight: { type: Number },
+    amount: { type: Number, required: true },
+    symbol: { type: String, required: true },
+    direction: { type: String, required: true },
+    txType: { type: String },
+    tipUrl: { type: String },
+    pending: { type: Boolean },
```

Comment:
> drilling down props to page because not sure where to lift up transactions getters, and avoid duplicated code between TransactionItem component and TransactionDetails page

extract some kind of action or getter to the store 🤷‍♀️